server.go

```diff
@@ -880,8 +880,9 @@ func SendHeader(ctx context.Context, md metadata.MD) error {
 }
 
 // SetTrailer sets the trailer metadata that will be sent when an RPC returns.
-// It may be called at most once from a unary RPC handler. The ctx is the RPC
-// handler's Context or one derived from it.
+// It may be called multiple times from a unary RPC handler.
+// When called more than once, all the provided metadata will be merged.
```

Comment:
How merging works is not important. The point is that all metadata will be sent, not just the last one.
What's the thing you find confusing here? If you think "merged" is not a good word, what about `all md's will be combined`? Or other suggestions?